docker/app/tests/test_checkpoint.py

```
@@ -0,0 +1,113 @@
+    'model_state_dict': model.state_dict(),
+    'optimizer_state_dict': optimizer.state_dict(),
+    'val_loss': best_loss,
+}
+
+torch.save(checkpoint, checkpoint_path)
+print(f"   ✓ チェックポイント保存完了: {checkpoint_path}")
+print(f"     - Epoch: {current_epoch}")
+print(f"     - Loss: {best_loss:.4f}")
+
+# 4. 新しいモデルとオプティマイザーを作成
+print("\n4. 新しいモデルとオプティマイザーを作成（リセット状態）")
```

Ruff (0.15.11)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)